core/frontend/src/render/webgl/PlanarClassifier.ts

```diff
@@ -345,7 +345,8 @@ export class PlanarClassifier extends RenderPlanarClassifier implements RenderMe
       return;
 
     // TBD - Refine resolution calculation -- increase height based on viewing angle.
-    const requiredHeight = Math.max(context.target.viewRect.width, context.target.viewRect.height);
+    // But make sure that we don't make it larger than the hardware supports, keeping in mind that we will be doubling the height later.
+    const requiredHeight = Math.max(Math.max(context.target.viewRect.width, context.target.viewRect.height), System.instance.capabilities.maxTextureSize / 2);

```

Comment:
This always selects `maxTextureSize / 2` except in unlikely case of `viewRect` being larger than that - is that intentional?

Reply:
Oops, no.  Good catch. This should have been a min.